whitepapers/business-value/content/en/business-drivers.md

```diff
@@ -0,0 +1,289 @@
+
+**In short, open source directly creates public value.**
+
+### Volunteers
+
+A lot of open source software is developed by volunteers. They do that to scratch an own itch, to learn, or to build up personal reputation. Motivations vary, and open source provides an environment to follow them while also offering the opportunity to make a big impact. This can be such a strong motivating factor that it brings together teams of developers which would not be possible to bring together by commercial means.
+
+Contributing voluntarily also gives a lot of freedom to follow your own interests and work in your own pace, because it doesn't come with the obligations of a paid assignment. This can open the room for experiments which would not be commercially viable.
+
+Many volunteers will even contribute to projects which serve as the base of a business of somebody else. They do so if they have the feeling that they get something in return, for example that the software they contribute to is well maintained and their efforts are multiplied.
+
+Volunteer contributions can be a delicate element of a business model. Volunteers owe nothing to a business exploiting their volunteer work, and unfair treatment of volunteers might not only harm a community but also seriously affect the reputation of a business. If done right, they can be a powerful multiplier of good will and a source of valuable contributions.
+
+While volunteers are the backbone of many open source projects, it's important to acknowledge the challenges of sustaining engagement. This can be especially taxing when a project is in high use with corresponding load and demands towards maintainers from commercial entities without compensating them adequately.
+
+**In short, respectful engagement with volunteers adds value beyond payroll.**
+
+## Business models
+
+In this section we look at concrete business models which use open source in some way to drive the business. We will use the drivers defined in the section above to explain how this is achieved.
+
+### Support and maintenance
+
+Selling commercial support and maintenance for open source projects. This includes guarantees for security updates and other services required for business to operate open source within an environment with service levels and similar requirements.
+
+Drivers:
+
+* Model directly scales with *adoption*. The more adoption, the more need for support and maintenance of these projects.
+* Model relies on *contributions*, so that fixes can be made on the timeline of the required support. To keep this economical they need to go back to the projects.
+* *Zero license costs* are a factor because range of support only depends on own ability to deliver not on the accessibility of the software supported. Scales well to large number of support offerings.
```

Comment:
This opens the market for other competitors to provide support (at least in theory) which makes it more attractive for customers compared to offerings where they are bound to a single vendor when choosing a specific solution and technology.